packages/block-editor/src/components/link-control/index.js

```diff
@@ -122,6 +122,7 @@ const PREFERENCE_KEY = 'linkControlSettingsDrawer';
  * @param {WPLinkControlProps} props Component props.
  */
 function LinkControl( {
+	searchInputLabel,
 	searchInputPlaceholder,
```

Comment:
Curious as to why we didn't just use `label` and `placeholder` here given that the `Text` field is an optional control.

Reply:
I think it would be to prevent confusion and specify what the label and placeholder are being used for
- The `label` and `placeholder` would be appropriate names for the `TextControl` component. 
- They might not be used as of now, but it would be easier in case it's added in the future 
- `searchInputLabel` and `searchInputPlaceholder` would suggest `props` for the Search Field Component